app/views/taxCodeDetails.scala.html

```diff
@@ -51,7 +50,31 @@
                     </p>
                 </div>
 
-                @multiDescriptionList(taxCodeViewModel.taxCodeDetails)
+                @for((descriptionList, descListIdx) <- taxCodeViewModel.taxCodeDetails zip (Stream from 1)) {
+
+                <section class="section" aria-labelledby="descriptionList@{descListIdx.toString}Heading">
+
+                    <h2 id="descriptionList@{descListIdx.toString}Heading">@descriptionList.heading</h2>
+
+                    <ul role="list" id="descriptionList@{descListIdx.toString}" class="tai-description-list flush" aria-labelledby="descriptionList@{descListIdx.toString}Heading">
+
+                        @for((descriptionTerm, termIdx) <- descriptionList.descriptionItems.keySet zip (Stream from 1)) {
```

Comment:
Should everything still be called description list since its no longer using that style?

Reply:
Yeah you're right - it no longer fits. I took a pragmatic decision not to change it, to minimise the time & impact, but since you've flagged it, I'll look at updating throughout.
The CSS classes also need to change, and the sass partial name itself - they all mention decsriptionList.